src/transformers/models/auto/tokenization_auto.py

```diff
@@ -207,7 +207,7 @@
                 ),
             ),
             ("openai-gpt", ("OpenAIGPTTokenizer", "OpenAIGPTTokenizerFast" if is_tokenizers_available() else None)),
-            ("opt", ("GPT2Tokenizer", None)),
+            ("opt", ("GPT2Tokenizer", "GPT2TokenizerFast" if is_tokenizers_available() else None)),
```

Comment:
Arf, dunno how this passed review!

Reply:
It was intentional at the time I think 😓

Reply:
Yes there was a misconception that it wasn't possible to get OPTFast written correctly when OPT was released.

The solution was definitely not obvious (since basically OPT uses "2" preprocessors, and there was no way to chain them). But one happened to be pretty much no-op, so there was a way to get it working.

However by caution it was decided to simply deactivate the Fast version of OPT.

All of this is linked to the fact that OPT, unlike GPT2 uses a BOS token at first.